master/.golangci.yml

```diff
@@ -31,23 +29,80 @@ issues:
     - Consider preallocating
     # Exclude "gosec: Errors unhandled" because it duplicates errcheck.
     - G104
+    - G601
     - and that stutters
     - declaration of "(err|ctx)" shadows declaration at
 
   # Independently from option `exclude` golangci-lint uses default exclude patterns.
   exclude-use-default: false
 
+  exclude-files:
+    - pkg/schemas/expconf/latest.go
+
   # Disable the maximum issue count per linter.
-  max-per-linter: 0
+  max-issues-per-linter: 0
 
 linters-settings:
   depguard:
-    list-type: blacklist
-    include-go-root: true
-    packages:
-      - errors
-      - gopkg.in/yaml.v2
-      - github.com/dgrijalva/jwt-go
+    rules:
+      main:
+        allow:
```

Comment:
What is this list for?

Reply:
why do we need an explicit allowlist now?

Reply:
to me the explicit allow is needing to run `go get package..` and having the resulting go.mod diff code reviewed.

Reply:
I can remove this then